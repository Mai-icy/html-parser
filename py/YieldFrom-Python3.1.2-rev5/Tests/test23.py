#
#   Test parsing yield from as function argument
#

from dis import dis

def g():
	f(yield from x)

dis(g)
