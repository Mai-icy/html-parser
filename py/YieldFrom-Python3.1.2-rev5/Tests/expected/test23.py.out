  8           0 LOAD_GLOBAL              0 (f) 
              3 LOAD_GLOBAL              1 (x) 
              6 YIELD_FROM           
              7 CALL_FUNCTION            1 
             10 POP_TOP              
             11 LOAD_CONST               0 (None) 
             14 RETURN_VALUE         
